keepassxc-browser/options/options.html

```diff
@@ -130,7 +130,12 @@ <h2>General Settings</h2>
             <label class="checkbox">
               <input type="checkbox" name="showNotifications" value="true" /> Show notifications.
             </label>
-            <span class="help-block">Allow KeePassXC-Browser to show notifications when user interaction is needed from the extension icon.</span>
+            <span class="help-block">Allow KeePassXC-Browser to show notifications about alerts and when user interaction is needed from the extension icon.</span>
```

Comment:
This is terribly worded. I have no idea what this setting is for after reading this.

Reply:
Maybe something like: "Show notifications on error situations or when user interaction is needed."?

Reply:
"Show notifications for errors and when user interaction is required"